api_tests/src/tags.spec.ts

```diff
@@ -95,6 +160,9 @@ test("Update post tags", async () => {
   });
   expect(postRes.post_view.post.id).toBeDefined();
 
+  // Wait post federated
+  await waitForPost(beta, postRes.post_view.post);
```

Comment:
At the moment both community and post are created on alpha so it doesnt fully test the federation. Change it to create a post from beta for better test coverage.